motivaa-backend/motivaa-app/src/test/java/backendtesting/TestUtils.java

```diff
@@ -0,0 +1,48 @@
+package backendtesting;
+
+import com.jayway.jsonpath.DocumentContext;
+import com.jayway.jsonpath.JsonPath;
+import org.testng.Assert;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class TestUtils {
+
+    public static List<Map<String,String>> retrieveLockVersionsFromResponseBody(String responseBody) {
+        return JsonPath.parse(responseBody).read("$.lockVersions");
+    }
+
+    public static String retrieveProcessUuidFromResponseBody(String responseBody) {
+        DocumentContext jsonContext = JsonPath.parse(responseBody);
+        return jsonContext.read("$.uuid");
+    }
+
+    public static void assertJsonPathValue(String responseBody,
+                                           String attributeFullTextIdPath,
+                                           String expectedValue,
+                                           String message) {
+
+        String jsonPath = String.format("$..components[?(@.id=='%s')].value", attributeFullTextIdPath);
+
+        List<String> jsonPathResultAsList = JsonPath.parse(responseBody).read(jsonPath);
+        if (jsonPathResultAsList.isEmpty()) {
+            Assert.fail("No value found for json path: " + jsonPath + " in response body: " + responseBody);
+        }
+        String jsonPathResult = jsonPathResultAsList.get(0);
+        Assert.assertEquals(
+                jsonPathResult,
```

Comment:
I think the first argument is the expected and the second the actual. You used it the other way around.

Reply:
I am using TestNg assert library (as testNg is the framework the codebase already had) and here it is actual first, expected second: https://www.javadoc.io/doc/org.testng/testng/6.8.17/org/testng/Assert.html

I have no idea why this is different then Junit assertion order, I was confused at first too.